package.json

```diff
@@ -36,11 +36,17 @@
     "@bigcommerce/stencil-paper": "^3.0.0-rc.29",
     "@bigcommerce/stencil-styles": "1.2.1",
     "@octokit/rest": "^18.0.3",
+    "@hapi/boom": "^8.0.1",
+    "@hapi/glue": "^6.2.0",
+    "@hapi/good": "^8.2.4",
```

Comment:
This is going to be deprecated soon. See https://www.npmjs.com/package/@hapi/good for details.
This is not a blocker but we should investigate and figure out if it is required or we drop the usage in a followup PR.

Reply:
Ok, I'll check it

Reply:
@junedkazi I checked, we can drop "@hapi/good" and "@hapi/good-console" since they aren't actually used in this project. I can create a corresponding followup PR when this one gets merged.